cli/src/schema/SchemaValidator.py

```diff
@@ -47,7 +47,11 @@ def validate_document(self, doc, schema):
     def run_for_individual_documents(self):
         for doc in self.validation_docs:
             # Load document schema
-            schema = load_schema_obj(types.VALIDATION, self.provider, doc.kind)
+            if 'backup' in doc.kind or 'recovery' in doc.kind:
+                schema = load_schema_obj(types.VALIDATION, self.provider, doc.kind)
```

Comment:
It looks like for `kind: configuration/backup` and `kind: configuration/recovery` schemas are exactly the same as before which was not working properly.